contentcuration/contentcuration/tests/test_exportchannel.py

```diff
@@ -1,116 +1,44 @@
-import md5
 import os
 import random
 import string
 import tempfile
-from cStringIO import StringIO
 
 import pytest
 from base import StudioTestCase
-from django.core.files.storage import default_storage
 from django.test.utils import override_settings
 from kolibri_content import models
 from kolibri_content.router import using_content_database
 from mixer.backend.django import mixer
 from mock import patch
 
+from .testdata import create_studio_file
+from .testdata import exercise
+from .testdata import slideshow
+from .testdata import topic
+from .testdata import video
 from contentcuration import models as cc
 from contentcuration.utils.publish import convert_channel_thumbnail
+from contentcuration.utils.publish import create_bare_contentnode
 from contentcuration.utils.publish import create_content_database
 from contentcuration.utils.publish import create_slideshow_manifest
-from contentcuration.utils.publish import create_bare_contentnode
 from contentcuration.utils.publish import MIN_SCHEMA_VERSION
 
 pytestmark = pytest.mark.django_db
 
 
-def video():
-    """
-    Create a video content kind entry.
-    """
-    return mixer.blend(cc.ContentKind, kind='video')
-
-
-def preset_video():
-    """
-    Create a video format preset.
-    """
-    return mixer.blend(cc.FormatPreset, id='mp4', kind=video())
-
-
-def topic():
-    """
-    Create a topic content kind.
-    """
-    return mixer.blend(cc.ContentKind, kind='topic')
-
-
-def exercise():
-    """
-    Create a topic content kind.
-    """
-    return mixer.blend(cc.ContentKind, kind='exercise')
-
-
-def slideshow():
-    """
-    Create a slideshow content kind.
-    """
-    return mixer.blend(cc.ContentKind, kind='slideshow')
-
-
-def preset_exercise():
-    """
-    Create an exercise format preset.
-    """
-    return mixer.blend(cc.FormatPreset, id='exercise', kind=exercise())
-
-
-def fileformat_perseus():
-    """
-    Create a perseus FileFormat entry.
-    """
-    return mixer.blend(cc.FileFormat, extension='perseus', mimetype='application/exercise')
-
-
-def fileformat_mp4():
-    """
-    Create an mp4 FileFormat entry.
-    """
-    return mixer.blend(cc.FileFormat, extension='mp4', mimetype='application/video')
-
-
-def license_wtfpl():
-    """
-    Create a license object called WTF License.
-    """
-    return mixer.blend(cc.License, license_name="WTF License")
-
```

Comment:
Above 60 line were line-for-line identical with those in `testdata.py` so removing --- I hope there was no reason to keep them sepearate.

Reply:
I think they were copied across from one to the other.